lnrpc/routerrpc/router_backend.go

```diff
@@ -55,6 +55,22 @@ type RouterBackend struct {
 	Tower routing.ControlTower
 }
 
+// MissionControl defines the mission control dependencies of routerrpc.
+type MissionControl interface {
```

Comment:
couldn't we define this interface in `routing`?

Reply:
We could, but these are `routerrpc`'s specific requirements for a mission control.

Reply:
yeah i think it makes sense to do it in `routerrpc`, so it can minimally define what it requires to operate